content/en/plugins/github-api/install/armory-cd.md

```diff
@@ -0,0 +1,137 @@
+---
+linkTitle: Armory CD
+title: Install the GitHub API Plugin in Armory Continuous Deployment
+weight: 1
+description: >
+  Learn how to install the GitHub API Plugin in Armory CD so that your app developers can trigger a Spinnaker pipeline from a GitHub workflow.
```

Comment:
`from a GitHub workflow.` can be rephrased as `when a Github workflow completes successfully`